tgui/packages/tgui/styles/components/Table.scss

```diff
@@ -22,6 +22,7 @@
 .Table__cell {
   display: table-cell;
   padding: 0 0.25em;
+  vertical-align: middle;
```

Comment:
Default alignment shouldn't be middle, but I see why you did this, because button loses its baseline due to your usage of flex. Although inline-flex should retain its baseline.